### PR TITLE
rename ArduinoLightMDNS to LightMDNS

### PR DESCRIPTION
**Pull request by matthewgream**
Taking Arduino out of the repository name as is not in the library name. Should be self explanatory.

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @matthewgream . I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2024/11/12 15:14:12 Loaded 7459 libraries from DB
Changing URL of library LightMDNS from https://github.com/matthewgream/ArduinoLightMDNS.git to https://github.com/matthewgream/LightMDNS.git
Success!
```

**Pull request merged**
